streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetadataState.java

```diff
@@ -349,8 +351,10 @@ private void rebuildMetadata(final Map<HostInfo, Set<TopicPartition>> activePart
                     final Map<String, Collection<String>> namedTopologyToStoreName = new HashMap<>();
                     final Set<String> topologyNames = topologyMetadata.namedTopologiesView();
                     topologyNames.forEach(topologyName -> {
-                        final Collection<String> storesOnHostForTopologyName = getStoresOnHost(storeToSourceTopics, activePartitionHostMap.get(hostInfo), topologyName);
-                        storesOnHostForTopologyName.addAll(getStoresOnHost(storeToSourceTopics, standbyPartitionHostMap.get(hostInfo), topologyName));
+                        final Map<String, List<String>> topologyStoresToSourceTopics =
```

Comment:
What's the difference between `topologyStoresToSourceTopics` and `storeToSourceTopics` here?